src/assets/BDDWithFlutter/BDDWithFlutter.md

```diff
@@ -423,7 +422,7 @@ index 068f558..5e0d8d0 100644
 
 ## Conclusion
 
-You have seen how to write Gherkin files and how to run them as automated tests for a flutter application.
+You have seen how to write Gherkin files and how to run them as automated tests for a Flutter application.
 I personally find flutter_gherkin a bit more complicated than other BDD frameworks, but it's possible, and I believe using BDD will improve the quality of your project greatly.
 
 If you need any help with the test-coverage of your app, BDD or other test-related topics, please contact us [@JankariTech](https://www.jankaritech.com)
```

Comment:
maybe add a note similar to this at the end

`This post was originally published on [insert date] and has been updated in July 2024.`

Reply:
I have created a PR https://github.com/JankariTech/blog/pull/152 to add `updatedAt` in metadata. If that gets merged then we don't need this otherwise I will add.